Python Samples/Plugin With Window/Plugin With Window.glyphsPlugin/Contents/Resources/plugin.py

```diff
@@ -0,0 +1,81 @@
+# encoding: utf-8
+
+###########################################################################################################
+#
+#
+#	General Plugin
+#
+#	Read the docs:
+#	https://github.com/schriftgestalt/GlyphsSDK/tree/master/Python%20Templates/General%20Plugin
+#
+#
+###########################################################################################################
+
+from __future__ import division, print_function, unicode_literals
+import objc
+from GlyphsApp import *
+from GlyphsApp.plugins import *
+import traceback
+
+
+class PluginWithWindow(GeneralPlugin):
+
+	"""
+	A GeneralPlugin has no windowcontroller.
```

Comment:
Den Kommentar eher dahin wo es darum geht an den WindowController zu gelangen.

Reply:
Updated and changed.
Also as proper python methods in that comment now. And